tests/pytest/lógica_de_asignacion/conftest.py

```diff
@@ -0,0 +1,124 @@
+'''
+Los fixtures que se definen acá están disponibles en todos los casos de prueba.
+
+Para usar un fixture, hay que ponerlo como argumento en la función del caso de
+prueba que lo necesita.
+
+Ver https://docs.pytest.org/en/stable/how-to/fixtures.html
+'''
+import pytest
+
+from ortools.sat.python.cp_model import CpModel
+from pandas import DataFrame
+import numpy as np
+
+from asignacion_aulica.lógica_de_asignación.dia import Día
+
+def pytest_configure(config):
+    # Registrar los markers usados por las fixtures
+    config.addinivalue_line("markers", "aulas")
+    config.addinivalue_line("markers", "clases")
+    config.addinivalue_line("markers", "asignaciones_forzadas")
```

Comment:
Ví [acá](https://docs.pytest.org/en/stable/example/markers.html#registering-markers) que se puede agregar una descripción de la marca en pytest.ini, probé y se puede hacer lo mismo con `addinivalue_line`, tal vez podemos aprovechar eso para describir el propósito de las marcas, por ejemplo:
```suggestion
    config.addinivalue_line( "markers", "aulas: marca para pasar parametros al fixture aulas")
    config.addinivalue_line( "markers", "clases: marca para pasar parametros al fixture clases")
    config.addinivalue_line( "markers", "asignaciones_forzadas: marca para pasar parametros al fixture asignaciones")
```